front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+
+import { Checkbox } from '@osrd-project/ui-core';
+import { ChevronDown, Clock, Flame, Manchette } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+
+import { Duration } from 'utils/duration';
+import { ms2min } from 'utils/timeManipulation';
+
+import TimetableItemActions from '../TimetableItemActions';
+import type { TrainScheduleWithDetails } from '../types';
+
+export type PacedTrain = TrainScheduleWithDetails & {
+  paced: {
+    duration: string;
+    step: string;
+  };
+};
+
+type PacedTrainItemProps = {
+  isInSelection: boolean;
+  handleSelectPacedTrain: (pacedTrainId: number) => void;
+  pacedTrain: PacedTrain;
+  isOnEdit: boolean;
+  isProjectionPathUsed: boolean;
+};
+
+const PacedTrainItem = ({
+  isInSelection,
+  handleSelectPacedTrain,
+  pacedTrain,
+  isOnEdit,
+  isProjectionPathUsed,
+}: PacedTrainItemProps) => {
+  const { t } = useTranslation(['operationalStudies/scenario']);
+
+  const [isOpened, setIsOpened] = useState(false);
```

Comment:
Maybe a bit long but we know what we handle here
```suggestion
  const [isOccurrencesListOpen, setIsOccurrencesListOpen] = useState(false);
```

Reply:
Done